### PR TITLE
ORBO videoSR App Helm Charts Uploaded 

### PR DESCRIPTION
**Pull request by amitorbo**

### CODE REVIEW

File: applications/orbovideosr/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{{/*
```
Comment (archie951) [inline applications/orbovideosr/templates/_helpers.tpl:1]: Can you please add the license in this file as well.

File: applications/orbovideosr/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+```text
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 ORBO
+```
+
+
+# **Orbo VideoSR Application Overview**
+Orbo Video Super Resolution converts a low resolution video into high resolution video.
```
Comment (archie951) [inline applications/orbovideosr/README.md:8]: Can you pack a little more information in overview. For example taking some literature from your website like "Cutting edge computer vision and deep learning technology is used to build 2X Super resolution algorithm which provides high definition images with improved details"

File: applications/orbovideosr/templates/_helpers.tpl (hunk 1 above)
Comment (abjyoti) [reply applications/orbovideosr/templates/_helpers.tpl:1]: Missing copyright header

**Review by abjyoti — CHANGES_REQUESTED** [review]
Resolve the comments

**Review by abjyoti — APPROVED** [review]

**Comment by abjyoti** [comment]
Approved for merging

**Pull request merged**